utils/paginated.js

```diff
@@ -0,0 +1,14 @@
+async function* paginated(perform, limit = Infinity) {
+  let nextPage = undefined;
+  for (let i = 0; nextPage != null, i < limit; i++) {
```

Comment:
I suggest to use logical `&&` operator instead of comma.

```suggestion
  for (let i = 0; nextPage != null && i < limit; i++) {
```

Reply:
Map returns undefined when there is no other page. That is why the script doesn't stop iterating once it reaches last page.

<img width="623" alt="image" src="https://user-images.githubusercontent.com/5206165/155146855-aa3ef80e-b1f3-497f-8370-e53774dd6af5.png">

Reply:
Oh, good catch.